english/quit-your-job/quit-your-job.md

```diff
@@ -1,2 +1,3 @@
 You suddenly decide that coding and enjoying your life are far mor important than saving money. 
-You decide that you are going to take the leap and quit your job in December 2015. 
+You decide that you are going to take the leap and quit your job in December 2015. Whoah, the time for making this change is fast 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.